apps/playground/src/app/components/components.module.ts

```diff
@@ -86,10 +89,12 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forChild(routes)],
+  imports: [RouterModule.forChild(componentRoutes)],
   exports: [RouterModule],
 })
-export class ComponentsRoutingModule {}
+export class ComponentsRoutingModule {
+  public static routes = componentRoutes;
```

Comment:
No. Because we lazy load the modules this isn’t possible. I didn’t quite a bit of research on this. The only thing returned from the router is the routes relevant to either the base or current route. These routes then have a `loadComponents` method that returns back the feature module type but not the inner routes. Putting that static property allows us to grab the information we need without having an instance of the module as that is what Angular gives us.